docs/outputs.rst

```diff
@@ -234,9 +346,8 @@ For CompCor decompositions, entries include:
 
 Confounds and "carpet"-plot on the visual reports
```

Comment:
This seems like it should maybe be a sub-heading under "confounds"? Just doesn't seem quite on the same level as the other 3 headers.